README

```diff
@@ -6,3 +6,16 @@ non-root users to view older versions of files and revert changes.
 
 For more information visit http://snapper.io/.
 
+Releasing
+=========
+
+Before releasing package ensure that changes made to package are mentioned
+in `package/snapper.changes`.
+To create package use commands `make -f Makefile.repo && make package`.
+Then use common work-flow to submit package to build service. For factory
+send at first package to devel project YaST:Head on OBS.
+
+When version is increased then git repo have to be tagged, obs project
+filesystems:snapper have to be updated and placing bzip2 on ftp.suse.com.
+When documentation changes e.g. man page or an important functionality, then
+also snapper.io webpages have to be updated.
```

Comment:
well, git tag is easy, updating OBS project is well documented.
Tricky parts are placing bzip2 where I am not sure what you need to achieve to get such permissions. For snapper.io I should probably add that you have to update gh-pages branch?